scripts/tests/run.sh

```diff
@@ -53,16 +53,12 @@ function test_init() {
   echo Superset init
   echo --------------------
   superset init
-  echo --------------------
-  echo Load examples
-  echo --------------------
-  pytest -s tests/load_examples_test.py
 }
 
 #
 # Init global vars
 #
-DB_NAME="test"
+DB_NAME="superset"
```

Comment:
This is not correct, the intend here is to use a separate DB and having a simple way to DROP/CREATE it at will